packages/dht/src/dht/DhtNode.ts

```diff
@@ -256,7 +256,7 @@ export class DhtNode extends EventEmitter<Events> implements ITransport {
             addContact: this.addNewContact.bind(this),
             connectionManager: this.connectionManager
         })
-        this.router = new Router({
+        this.routerRpcLocal = new RouterRpcLocal({
```

Comment:
This is not necessarily a good renaming since we need to bring back a class called router. RouterRpcLocal should not be the class which handles the routing logic, it should only have the local rpc methods bound to the Rpc server

Reply:
Reverted back to `Router`. Let's extract the  LocalRpc functionality in a separate PR.